### PR TITLE
fixed cors

### PR DESCRIPTION
**Pull request by dalidia**

### CODE REVIEW

File: server/run.py [hunk 1]
```diff
@@ -65,16 +66,19 @@ def load_user(user_id):
...
-        response.headers.add(
-            "Access-Control-Allow-Headers", "Content-Type,Authorization,Set-Cookie"
-        )
-        response.headers.add("Access-Control-Allow-Methods", "GET,PUT,POST,DELETE")
-        response.headers.add("Access-Control-Allow-Credentials", "true")
-        response.headers.add("Access-Control-Expose-Headers", "X-User-Id")
-        if current_user.is_authenticated:
-            response.headers.add("X-User-Id", f"{current_user.id}")
-        return response
+        allowed_origins = [f"{FRONT_END_HOST}", "https://frontend404.herokuapp.com", "https://website404.herokuapp.com", "https://backend-404.herokuapp.com"]
```
Comment (giancarlopernudisegura) [inline server/run.py:70]: Instead of hardcoded values, we should use the values from the remote node DB table.

**Review by giancarlopernudisegura — CHANGES_REQUESTED** [review]

File: server/run.py (hunk 1 above)
Comment (dalidia) [reply server/run.py:70]: can you do that? I have no idea how the DB has been set up

Comment (giancarlopernudisegura) [reply server/run.py:70]: yeah sure

Comment (giancarlopernudisegura) [reply server/run.py:70]: Yeah gonna leave this for after the deadline, putting a TODO for now.

**Review by giancarlopernudisegura — APPROVED** [review]

**Pull request merged**